### DISCUSSION TITLE
vcpkg no longer reports versions on errors

### DISCUSSION THREAD

**Discussion by dg0yt** [microsoft/vcpkg#31389] [Q&A]
**To Reproduce**
1. install a failing port (e.g. readline-win32 on x64-linux)
2. Version information is printed.

Check recent issues for examples.
e.g. #30787

**Expected behavior**
There used to be:
~~~
    vcpkg-tool version: 2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75
    vcpkg-scripts version: 1c48ec7 2023-02-07 (10 days ago)
~~~

e.g. #29725

**Comment by autoantwort** [comment]
This is intentional. The idea was to make the large error block shorter and motivate the people to use the auto generated issue body which still contains these information.

**Comment by MonicaLiu0311** [comment]
Sorry, this is a by design; if there are no other problems, this issue will be closed.

**Comment by dg0yt** [comment]
You should evaluate if the change worked good enough. AFAICS it works only for people that do use the GH CLI.

I wouldn't mind if the information was printed at the beginning of the install process. Most user provide the complete console log.

**Comment by dg0yt** [comment]
> I wouldn't mind if the information was printed at the beginning of the install process.

More elaborated:
- Tool version before all ports.
- Port version at the beginning of the installation.
  (In that case, the repo version might be omitted if vcpkg wants to avoid the call to `git`. Problems from dependencies are connected to their installed versions, not to the available versions.)
